troubleshooting.md

```diff
@@ -726,3 +726,21 @@ And then re-add the connection (removing the old one if necessary):
 And now this should work:
 
 `podman-remote info`
+
+---
+### 28) CNI networking fails in Podman v2.2.1 and earlier.
+
+A failure is encountered when trying to use networking on a rootless
+container in Podman v2.2.1 or earlier.  This version of Podman was included in
+RHEL 8.3.1.
+
+#### Symptom
+
+Two rootless containers are created on the host but are not able to communicate
+with each other via their virtual network devices.
+
+#### Solution
+
+In order to use a CNI network in a rootless container in Podman v2.2.1 and earlier,
+an Infra container image for CNI-in-slirp4netns must be created.  The
+instructions for building the Infra container image can be found [here](https://github.com/containers/podman/tree/v2.2.1-rhel/contrib/rootless-cni-infra).
```

Comment:
This is only applicable to v2.2.1. The other version we shipped on RHEL with this functionality, v3.0.1, requires a newer version of the image from https://github.com/containers/podman/tree/v3.0.1-rhel/contrib/rootless-cni-infra

Generally speaking, it's a hard requirement to match the image version to the Podman version, but since non-RHEL Podman can only pull the image and cannot have it manually built, we only need to mention these two.